CMakeLists.txt

```diff
@@ -70,7 +70,11 @@ ENDMACRO(ADD_TESTS)
 
 ADD_SUBDIRECTORY( src )
 ADD_SUBDIRECTORY( doc )
-ADD_SUBDIRECTORY( tests EXCLUDE_FROM_ALL )
+IF ( ENABLE_BUILD_TESTS )
+  ADD_SUBDIRECTORY( tests )
+ELSE ( ENABLE_BUILD_TESTS )
+  ADD_SUBDIRECTORY( tests EXCLUDE_FROM_ALL )
+ENDIF ( ENABLE_BUILD_TESTS )
```

Comment:
`OPTION (ENABLE_BUILD_TESTS "Build and run test suite by default?" OFF)`
Maybe this should be added to the OPTION section.

Reply:
Done